zero_bin/rpc/Cargo.toml

```diff
@@ -7,6 +7,7 @@ license.workspace = true
 repository.workspace = true
 keywords.workspace = true
 categories.workspace = true
+build = "build.rs"
```

Comment:
You could probably put one `build.rs` to root of `zero_bin` and point it from the 'leader', `rpc`, `worker`, `verifier` like `build = "version.rs"`. i believe it is fine to put `EVM_ARITHMETIZATION_PACKAGE_VERSION` for all binaries, so using leader `build.rs` for all is OK I think.